core/pom.xml

```diff
@@ -398,7 +398,7 @@ THE SOFTWARE.
     <dependency><!-- groovy shell uses this but uses an optional dependency. Not included into BOM -->
       <groupId>jline</groupId>
       <artifactId>jline</artifactId>
-      <version>2.12</version>
+      <version>2.14.6</version>
```

Comment:
apparently unmaintained and has become jline3:
https://github.com/jline/jline3

probably worth a look to see if it's compatible